airgun/entities/host.py

```diff
@@ -295,6 +300,9 @@ def step(self, *args, **kwargs):
             raise ValueError('Please provide a valid action name.'
                              ' action_name: "{0}" not found.')
         entities_list = kwargs.get('entities_list')
-        for entity in entities_list:
-            self.parent.table.row(name=entity)[0].widget.fill(True)
+        if entities_list == "All":
+            self.parent.select_all.click()
```

Comment:
Nitpick - please follow widgetastic conventions:
```suggestion
            self.parent.select_all.fill(True)
```